EpsilonAuxBmsV2/Core/Src/Tasks/OrionInterfaceTask.c

```diff
@@ -62,13 +62,13 @@ void orionInterface(OrionCanInfo* message)
     {
         // Determine trip conditions and contactor settings based on Orion CAN
         localAuxStatus.orionCanReceivedRecently = 1;
-        checkCellVoltage(message, &localAuxStatus);
-        localAuxStatus.dischargeShouldTrip = checkDischargeTrip(message, &localAuxTrip);
-        localAuxStatus.chargeShouldTrip = checkChargeTrip(message, &localAuxTrip);
-        uint8_t protectionTrip = checkProtectionTrip(message, &localAuxTrip); // Without this line, the test_TooHighMaxCellVoltageShouldTurnOffChargeAndUpdateStatuses test fails for some reason
-        shouldDisconnectContactors = localAuxStatus.dischargeShouldTrip
-                                     || localAuxStatus.chargeShouldTrip
-                                     || protectionTrip;
+        updateAllowChargeAndAllowDischarge(message, &localAuxStatus);
+        updateAuxTrip(message, &localAuxTrip);
+        localAuxStatus.dischargeShouldTrip = checkDischargeTrip(message);
+        localAuxStatus.chargeShouldTrip = checkChargeTrip(message);
+        shouldDisconnectContactors = checkProtectionTrip(message)
```

Comment:
I think you can just use `localAuxStatus.protectionTrip` here

Reply:
Instead of `shouldDisconnectContactors`?

Reply:
I mean I think you already set up `localAuxStatus.protectionTrip` in `updateAuxTrip`, so you could just use that instead of a new function?

Reply:
I can't remember exactly, but Bill might be right. If we're updating `localAuxStatus.protectionTrip` before this, then we can just use that instead.

Reply:
![image](https://user-images.githubusercontent.com/56768212/109245034-0442f580-779d-11eb-8a37-21a784a7d3d3.png)
I do not see `protectionTrip` in `AuxStatus`.

Reply:
oooooooh sorry, I meant `localAuxTrip`, got confused between the two lol

But yeah `updateAuxTrip` sets `localAuxTrip.protectionTrip`, so that can be used in the check